server/store/sqlstore/migrations.go

```diff
@@ -17,6 +17,18 @@ func (s *SQLStore) runMigrationRemoteID(remoteID string) error {
 	return err
 }
 
+func (s *SQLStore) runSetEmailVerifiedToTrueForRemoteUsers(remoteID string) error {
```

Comment:
Not an issue per se, but this will run on every startup since there's no short-circuit or negation switch. 

(Looking forward to the day we have sequenced/conditional migrations!)

Reply:
Yeah, that's why the condition has the `RemoteID=xx AND Emailverified == false`, in theory, the resultset with those conditions should always be 0... except for the first run.